source/manual/govuk-notify.html.md

```diff
@@ -33,6 +33,15 @@ for each environment):
   example might be to receive an email when a scheduled document on GOV.UK has
   been automatically published.
 
+Currently (as of 30/06/2020) GOV.UK Notify has a maximum rate limit of 350
+requests per second meaning we should adhere to this limit when using Notify to
+deliver emails for our applications. An example of where we set this for
+EmailAlertApi can be found in the [DeliveryRequestWorker][DeliveryRequestWorker].
+
+If you wish to double check this figure you could ask in their slack channel #govuk-notify.
+
+[DeliveryRequestWorker]: https://github.com/alphagov/email-alert-api/blob/master/app/workers/delivery_request_worker.rb#L52
+
 _In the future we may set up a new 'GOV.UK Public' service which is used to
```

Comment:
Out-of-scope, but can we remove this line? It's really strange to see a "idea" in our developer docs.